includes/languages/lang.english.php

```diff
@@ -529,6 +529,7 @@
     'WARNING_NO_FILE_UPLOADED' => 'Warning:  no file uploaded.',
     'WARNING_PRODUCT_QUANTITY_ADJUSTED' => 'Quantity has been adjusted to what is in stock. ',
     'WARNING_SHOPPING_CART_COMBINED' => 'NOTICE: For your convenience, your current shopping cart has been combined with your shopping cart from your last visit. Please review your shopping cart before checking out.',
+    'WHOLESALE_TAX_EXEMPT_DESCRIPTION' => 'Wholesale Customer: Tax Exempt',
```

Comment:
Since this could appear as a tax-description in an order's order-totals' display, please simplify to remove the intervening `:` since the display would wind up looking like
```
Wholesale Customer: Tax Exempt:  $0.00
```

I'll suggest something akin to `Tax Exempt (Wholesale)`.

Reply:
i have renamed and removed colon.

you do not like my name, feel free to change it.